app/controllers/TransferAgreementController.scala

```diff
@@ -12,7 +12,8 @@ import play.api.data.Form
 import play.api.data.Forms._
 import play.api.i18n.{I18nSupport, Lang, Langs}
 import play.api.mvc.{Action, AnyContent, Request, Result}
-import services.GetConsignmentService
+import services.ConsignmentService
+import uk.gov.nationalarchives.tdr.error.NotAuthorisedError
```

Comment:
This import doesn't appear to work when I try this locally?

Reply:
Do you have the latest version? It's in version 0.0.12.